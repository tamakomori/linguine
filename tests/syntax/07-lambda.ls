func main() {
    f = lambda (a) => { return a; };
    print("f(1) = " + f(1));

    g = lambda (a, b) => { return a + b; };
    print("g(1, 2) = " + g(1, 2));
}
