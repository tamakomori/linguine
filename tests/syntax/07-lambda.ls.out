f(1) = 1
g(1, 2) = 3
